docs/hub/config.md

```diff
@@ -42,7 +42,8 @@ When using environment variables, list must be space separated. As flags paramet
 | `write_timeout`                  | maximum duration before timing out writes of the response, set to `0s` to disable (default), example: `2m`                                                                                                                                                                                                                                                                                                                                                       |
 | `use_forwarded_headers`          | set to `true` to use the `X-Forwarded-For`, and `X-Real-IP` for the remote (client) IP address, `X-Forwarded-Proto` or `X-Forwarded-Scheme` for the scheme (http or https), `X-Forwarded-Host` for the host and the RFC 7239 `Forwarded` header, which may include both client IPs and schemes. If this option is enabled, the reverse proxy must override or remove these headers or you will be at risk                                                        |
 | `dispatch_subscriptions`         | set to `true` to dispatch updates when a subscription between the Hub and a subscriber is established or closed. The topic follows the template `https://mercure.rocks/subscriptions/{subscriptionID}`. To receive connection updates, subscribers must have `https://mercure.rocks/targets/subscriptions` or an URL matching the template `https://mercure.rocks/targets/subscriptions/{topic}` (`{topic}` is URL-encoded topic of the subscription) as targets |
-| `subscriptions_include_ip`       | set to `true` to include the subscriber's IP in the subscription update                                                                                                                                                                                                                                                                                                                                                                                           |
+| `subscriptions_include_ip`       | set to `true` to include the subscriber's IP in the subscription update                                                                                                                                                                                                                                                                                                                                                                                          |
+| `metrics`                        | set to `true` to enable the `/metrics` HTTP endpoint. Provide metrics for Hub monitoring                                                                                                                                                                                                                                                                                                                                                                         |
```

Comment:
```suggestion
| `metrics`                        | set to `true` to enable the `/metrics` HTTP endpoint. Provide metrics for Hub monitoring in the OpenMetrics format |
```

Reply:
I also wonder if we shouldn't register this handler in the `addHealthkeck)` function to bypass the HTTP logger and other middleware? It will improve the performance and prevent log pollution. WDYT?